files/default/configure_nw_interface.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# expected vars:
+# DEVICE_NAME: name of the device
+# DEVICE_NUMBER: number of the device
+# GW_IP_ADDRESS: gateway ip address
+# DEVICE_IP_ADDRESS: ip address to assign to the interface
+# CIDR_PREFIX_LENGTH: the prefix length of the device IP cidr block
+# NETMASK: netmask to apply to device ip address
```

Comment:
Let's add a chefk for all the required variables.
If one variable is missing we can configure the network interfaces with broken values.

Let's add something like:
```
if [ -z ${DEVICE_NAME} ]; then
  echo "DEVICE_NAMEmust be set"
  exit 1
elif [ -z ${DEVICE_NUMBER} ]; then
  echo "DEVICE_NUMBERmust be set"
  exit 1
elif ...
```